lib/docblockr-worker.js

```diff
@@ -803,6 +803,15 @@ module.exports =
         i++;
       }
       return a;
+    }; 
+    
+    DocBlockrAtom.prototype.get_relevant_editor = function() {
+        var editor = atom.workspace.getActiveTextEditor();
+        if (editor !== undefined)
+            return editor;
+        if (this.event && this.event.target)
+            return this.event.target.getModel();
+        return null;
```

Comment:
Thanks! Always good to validate assumptions 😅  thanks also for the pointer on how to implement!